data-pipeline.yml

```diff
@@ -0,0 +1,211 @@
+version: 1
+tasks:
+  - ID: commit-retrieval
+    created: {$fromNow: ''}
+    deadline: {$fromNow: '12 hours'}
+    provisionerId: aws-provisioner-v1
+    workerType: release-svc-compute
+    payload:
+      maxRunTime: 86400
+      image: mozilla/bugbug-commit-retrieval
+
+      artifacts:
+        public/commits.json.xz:
+          path: /data/commits.json.xz
+          type: file
+
+    metadata:
+      name: bugbug commit retrieval
+      description: bugbug commit retrieval
+      owner: release-mgmt-analysis@mozilla.com
+      source: https://github.com/mozilla/bugbug/raw/master/data-pipeline.yml
+
+  - ID: bugs-retrieval
+    created: {$fromNow: ''}
+    deadline: {$fromNow: '1 hour'}
+    provisionerId: aws-provisioner-v1
+    workerType: releng-svc
+    payload:
+      maxRunTime: 86400
+      image: mozilla/bugbug-bugs-retrieval
+
+      artifacts:
+        public/bugs.json.xz:
+          path: /data/bugs.json.xz
+          type: file
+
+      features:
+        taskclusterProxy:
+          true
+
+    metadata:
+      name: bugbug bugs retrieval
+      description: bugbug bugs retrieval
+      owner: release-mgmt-analysis@mozilla.com
+      source: https://github.com/mozilla/bugbug/raw/master/data-pipeline.yml
+
+  - ID: train-component
+    created: {$fromNow: ''}
+    deadline: {$fromNow: '1 hour'}
```

Comment:
Some of the train task will take longer than 1 hour (this one probably around 3?). We could use 7 for all of them, and then update it when we have more data on how long they take.